src/Command/Docker/GenerateDist.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Command\Docker;
+
+use Magento\MagentoCloud\Docker\Config\Dist\Generator;
+use Magento\MagentoCloud\Docker\ConfigurationMismatchException;
+use Magento\MagentoCloud\Filesystem\FileSystemException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Generates .dist files.
+ */
+class GenerateDist extends Command
+{
+    const NAME = 'docker:generate-dist';
+
+    /**
+     * @var Generator
+     */
+    private $distGenerator;
+
+    /**
+     * @param Generator $distGenerator
+     */
+    public function __construct(Generator $distGenerator)
+    {
+        $this->distGenerator = $distGenerator;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
You should use `{}` only if you expand the description.

For example:
```
/**
 * @inheretDoc
 */
```

```
/**
 * {@inheretDoc}
 * @return bool
 */
```

Reply:
Please, fix all places with `@inheritDoc`

Reply:
Also, I see that usually, we use `@inheritdoc`, you write `@inheritDoc`
I think we should use one style.

Reply:
There is no clear explanation of `@inheritDoc`, until the clear explanation of this tag, I don't see a big reason to point attention on it.